src/components/product/ProductsList.component.tsx

```diff
@@ -0,0 +1,12 @@
+import { products } from '@/api/products.ts';
+import { ProductCardComponent } from '@/components/product/ProductCard.component.tsx';
```

Comment:
better use the same name for component and file name for all components
your: usage: `import { ProductCardComponent } from '@/components/product/ProductCard.component.tsx';`
better: usage: `import { ProductCard } from '@/components/product/ProductCard.component.tsx';`